node-setup/000_install.md

```diff
@@ -30,7 +30,7 @@ If we are using an AWS instance running Amazon Linux AMI 2 (see the [AWS walk-th
 
     sudo yum update -y
     sudo yum install git gcc gcc-c++ tmux gmp-devel make tar wget zlib-devel -y
-    sudo yum install systemd-devel ncurses-devel ncurses-compat-libs -y
+    sudo yum install systemd-devel ncurses-devel ncurses-compat-libs libsodium-devel -y
```

Comment:
This didn't work out of the box for me, I had to add another source for the package, as it wasn't available at aws linux 2. 

This is what I had to do:
First add the lux repo to yum:

```
sudo rpm -Uvh http://repo.iotti.biz/CentOS/6/noarch/lux-release-6-3.noarch.rpm
sudo rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-LUX
```

I then had to edit the .repo file to point to the right place for some reason. edit this file /etc/yum.repos.d/lux.repo to contain the following:

```
[lux]
name=CentOS $releasever - $basearch - Lux
baseurl=http://repo.iotti.biz/CentOS/6
enabled=1
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-LUX  
```

[source](https://stackoverflow.com/a/60650050/2552259)